docs/markdown/Reference-manual.md

```diff
@@ -1129,6 +1129,32 @@ This function has one keyword argument.
    recurse in the subdir if they all return `true` when queried with
    `.found()`
 
+### subdir_done()
+
+``` meson
+    subdir_done()
+```
+
+Stops further interpretation of the meson script file from the point of
+the invocation. All steps executed up to this point are valid and will
+be executed by meson. This means that all targets defined before the call
+of `subdir_done` will be build.
```

Comment:
If you want a paragraph break here, add one more newline (an empty line). If not, rewrap the text to not have this extra newline.